src/coreclr/src/System.Private.CoreLib/src/System/ArgIterator.cs

```diff
@@ -85,7 +85,9 @@ public TypedReference GetNextArg(RuntimeTypeHandle rth)
                 // malicious caller to increment the pointer to an arbitrary
                 // location in memory and read the contents.
                 if (ArgPtr == IntPtr.Zero)
+#pragma warning disable CA2208 // Instantiate argument exceptions correctly, the argument not applicable
```

Comment:
The error is not related to argument, might supposed to use IOException, suppressing for now